code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -51,6 +51,13 @@ var/list/hardcoded_gases = list("o2","n2","co2","plasma") //the main four gases,
 	gas_overlay = "plasma"
 	moles_visible = MOLES_PLASMA_VISIBLE
 
+/datum/gas/freon
+	id = "freon"
+	specific_heat = 1
```

Comment:
Setting this to 1 is a terrible idea. Make it around 2000 and just have the gas start cold.